lib/host.rb

```diff
@@ -0,0 +1,54 @@
+require "./lib/order.rb"
+require "./lib/menu.rb"
+require "Dotenv"
+require "twilio-ruby"
+
+
+class Host
+
+  attr_reader :current_menu, :new_order
+
+  def initialize(current_menu = Menu.new, order_klass = Order)
+    @credentials = Dotenv.load
+    @current_menu = current_menu
+    @order_klass = order_klass
+    @new_order = @order_klass.new
+    @client = Twilio::REST::Client.new @credentials["ACCOUNT_SID"], @credentials["AUTH_TOKEN"]
```

Comment:
Metrics/LineLength: Line is too long. [94/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)